src/pip/_internal/utils/misc.py

```diff
@@ -375,22 +373,6 @@ def write_output(msg: Any, *args: Any) -> None:
     logger.info(msg, *args)
 
 
-class StreamWrapper(StringIO):
```

Comment:
This is no longer required as https://github.com/python/cpython/issues/88832 was fixed in Python 3.9+.